lib/src/common/topic.dart

```diff
@@ -38,6 +41,14 @@ class Topic {
   static String userPrivateStoreKeyBundle(String walletAddress) =>
       _content('privatestore-${_normalize(walletAddress)}/key_bundle');
 
+  static Future<String> userPreferences(List<int> privateKey) async =>
+      _content('pppp-${await generatePPPPIdentifier(privateKey)}');
```

Comment:
We renamed this to be more descriptive it is now 
```suggestion
      _content('userpreferences-${await generatePPPPIdentifier(privateKey)}');
```